actionpack/lib/action_controller/railtie.rb

```diff
@@ -24,7 +24,7 @@ class Railtie < Rails::Railtie #:nodoc:
     initializer "action_controller.parameters_config" do |app|
       options = app.config.action_controller
 
-      ActiveSupport.on_load(:action_controller) do
+      ActiveSupport.on_load(:action_controller_base) do
```

Comment:
Know I see @rafaelfranca, the first time the hook is executed [this line](https://github.com/rails/rails/blob/master/actionpack/lib/action_controller/railtie.rb#L33) deletes the value for `:action_on_unpermitted_parameters` and the next time it gets executed `options[:action_on_unpermitted_parameters]` is `nil` and the value gets overwritten.

I will correct the PR accordingly.